root/static/scripts/edit/MB/Control/Area.js

```diff
@@ -29,4 +29,4 @@ MB.Control.Area = function () {
     });
 };
 
-module.exports = MB.Control;
+export default MB.Control;
```

Comment:
There's an import in root/static/scripts/place.js which this breaks.

Perhaps replace this line with something like `export const initializeArea = MB.Control.Area;` and change the import to `import {initializeArea} from './edit/MB/Control/Area';`.